src/lib/Lib/Config.cpp

```diff
@@ -198,6 +199,12 @@ struct PublicSettingsVisitor {
         else if (referenceDirKey == "mrdocs-root") {
             return dirs.mrdocsRoot;
         }
+        else if (referenceDirKey == "output") {
+            if (settings.output.empty()) {
```

Comment:
`MRDOCS_ASSERT` is a pre-condition the developer needs to fix if it's not met. The user has nothing to do with it.